cmd/main.go

```diff
@@ -72,9 +73,16 @@ func main() {
 			if len(args) != 3{
 				return errors.New("Wrong number of Input arguments expected 3 got "+strconv.Itoa(len(args)))
 			}
-			if (tlsConn == false && insecure == true) {
+			if (args[1][0:5] != "amqps" && insecure) {
 				return errors.New("Cannot have an insecure TLS connection without TLS enabled")
 			}
+			if (args[1][0:5] != "amqps" && tlsPath != "") {
```

Comment:
I think it is possible to have TLS without a cert. I would expect that in this case the system defaults will get used.